.github/workflows/tests.yml

```diff
@@ -0,0 +1,43 @@
+name: Open Source Programs Run Tests
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: Run tests
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres
+        env:
+          POSTGRES_USER: osp
+          POSTGRES_PASSWORD: osp
+          POSTGRES_DB: osp
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          - 5432:5432
+    
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.7
+    
+    - name: Installing dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r requirements.txt
+    
+    - name: Creating Zulip API key
+      run: echo -e "${{ secrets.ZULIP_API_KEY }}" >> download
```

Comment:
Why is this needed for tests @codesankalp ?

Reply:
The test will not perform without this. Because to perform test there must be no error in code.
And in `utils/zulip_api.py` we are using the `download` file as a config file for Zulip Stats which is a must needed condition before running any command @isabelcosta

Reply:
Ok, I will approve this. But we have to find a way to not need real API KEY to test, by mocking zulip response when needed, or in last scenario if really needed get a zulip test account. I think we need an issue for this